backend/src/packages/projects/project.repository.ts

```diff
@@ -17,11 +19,18 @@ class ProjectRepository
     return projects.map((project) => ProjectEntity.initialize(project));
   }
 
-  public async findByUserId(id: number): Promise<ProjectEntity[]> {
+  public async findByUserId(
+    { query }: ProjectSearchParameters,
+    id: number,
```

Comment:
I believe first parameter should be the main filtering option, and second one the optional filters
```suggestion
    id: number,
    { query }: ProjectSearchParameters,
```